.github/workflows/cargo_update_check.yml

```diff
@@ -16,6 +16,8 @@ jobs:
 
       - name: Update and Check Ockam Example
         working-directory: examples/rust/get_started
+        env:
```

Comment:
This is done in the env because `cargo check` doesn't accept `-Dwarnings` as an argument — cargo clippy does, which this doesn't use)